src/AzurePipelines/PipelineCachingCacheClient.cs

```diff
@@ -292,12 +292,22 @@ protected override async Task AddNodeAsync(
                 result.ProofNodes,
                 ContentFormatConstants.Files);
 
-            CreateResult createResult = await WithHttpRetries(
-                () => _cacheClient.CreatePipelineCacheArtifactAsync(entry, null, cancellationToken),
+            await WithHttpRetries(
+                async () =>
+                {
+                    try
+                    {
+                        CreateResult createResult = await _cacheClient.CreatePipelineCacheArtifactAsync(entry, null, cancellationToken);
+                        Tracer.Debug(context, $"Cache entry for {fingerprint} stored in scope `{createResult.ScopeUsed}`");
+                    }
+                    catch (PipelineCacheItemAlreadyExistsException)
+                    {
+                        Tracer.Debug(context, $"Cache entry for {fingerprint} already exists.");
```

Comment:
What about other implementations?

Reply:
This change matches the behavior of the others: https://github.com/microsoft/MSBuildCache/blob/ce365612ae6c526bd235d7179b38246c10d1ac94/src/Common/Caching/CasCacheClient.cs#L245